ext/standard/tests/time/idate.phpt

```diff
@@ -3,29 +3,31 @@ idate() function
 --FILE--
 <?php
 date_default_timezone_set('GMT0');
-$tmp = "UYzymndjHGhgistwLBIW";
+$tmp = "UYozymndjHGhgistwNLBIW";
```

Comment:
Please don't change existing tests, but add a new one. Especially because you changed the timestamp, which IMO is a bad idea.

Reply:
Why is that?